### PR TITLE
Blue note: single templates improvements

### LINKED ISSUE #7: [Blue Note]: Single template with no sidebar
**Desktop:**

Single (no sidebar)

**Mobile:**

Single (no sidebar) - Mobile

### PR DESCRIPTION
**Pull request by MaggieCabrera**
This PR fixes this issues with both single templates:

- [x] Turns sidebar into a pattern for translation
- [x] Adds comments and post meta to the single no sidebar template
- [x] Removes numbers from comments pagination
- [x] Correct font weight for post titles on post navigation
- [x] Corrects color and size of separator block

Closes https://github.com/WordPress/community-themes/issues/7 and https://github.com/WordPress/community-themes/issues/6

### CODE REVIEW

File: blue-note/patterns/post-meta-nav.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		<hr class="wp-block-separator has-alpha-channel-opacity"/>
 		<!-- /wp:separator -->
 		<!-- wp:post-navigation-link {"type":"previous","label":"<em><?php esc_html_e( 'Previous', 'blue-note' ); ?></em>","linkLabel":true,"arrow":"arrow"} /-->
-		<!-- wp:post-navigation-link {"type":"previous","label":"","showTitle":true,"style":{"typography":{"textTransform":"uppercase"}}} /-->
+		<!-- wp:post-navigation-link {"type":"previous","label":"","showTitle":true,"style":{"typography":{"textTransform":"uppercase","fontWeight":"600"}}} /-->
```
Comment (scruffian) [inline blue-note/patterns/post-meta-nav.php:35]: Should we make these changes in theme.json?

**Review by scruffian — APPROVED** [review]
LGTM. One question but I think it's debateable.

File: blue-note/patterns/post-meta-nav.php (hunk 1 above)
Comment (MaggieCabrera) [reply blue-note/patterns/post-meta-nav.php:35]: If we do then we need to make the other instance of it right above not bold :D

File: blue-note/patterns/sidebar-content.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+	<!-- wp:heading {"style":{"typography":{"fontSize":"22px","textTransform":"uppercase","fontWeight":"700"}}} -->
+	<h2 class="wp-block-heading" style="font-size:22px;font-weight:700;text-transform:uppercase"><?php esc_html_e( 'Categories', 'blue-note' ); ?></h2>
+	<!-- /wp:heading -->
+	<!-- wp:categories {"showPostCounts":true} /-->
+</div>
+<!-- /wp:group -->
```
Comment (carolinan) [inline blue-note/patterns/sidebar-content.php]:
```suggestion
<!-- /wp:group -->

```
Adds an empty new line

**Review by carolinan — APPROVED** [review]

**Pull request merged**